ddp-workspace/projects/ddp-dsm-ui/src/app/FON/pages/participantsList/participantsList.component.ts

```diff
@@ -42,8 +49,8 @@ export class ParticipantsListComponent implements OnInit {
         tap((params: Params) => {
           this.pageSize = params.to - params.from;
           this.pageIndex = (params.to/this.pageSize) - 1;
-        }),
-        switchMap((params: Params) => this.agent.setPage(params.from, params.to))
+          this.patients$ = this.storeService.getParticipants(params.from, params.to, this.PARENT);
```

Comment:
This is an example of dispatching action from Store service in order to trigger changes in the store.